src/index.css

```diff
@@ -1,3 +1,4 @@
+/* @import url('../node_modules/document-model-libs/dist/style.css'); */
```

Comment:
probably can delete this comment

Reply:
removed here [6da080c](https://github.com/powerhouse-inc/document-model-electron/pull/87/commits/6da080c53ea57352b997de941fc303f11a5d2a42)